tests/queries/0_stateless/01062_pm_all_join_with_block_continuation.sql

```diff
@@ -21,11 +21,12 @@ SELECT count(1), uniqExact(n) FROM (
     JOIN (SELECT materialize(1) AS k, number n FROM numbers(10)) j
     USING k);
 
--- errors
 SET max_joined_block_size_rows = 0;
 
 SET query_plan_join_swap_table = 'false';
 
+-- Because of the optimizations in the analyzer the following queries started to run without issues. To keep the essence of the test, we test both cases.
+SET enable_analyzer = 0;
```

Comment:
afaiu, we have separate runs with and without analyzer in CI. So, there should be no need to change the test